.github/workflows/build-init.yml

```diff
@@ -0,0 +1,30 @@
+name: CI-build
+
+on:
+  pull_request:
+    paths-ignore: ['docs/**']
```

Comment:
@ltalirz I've added this build job to the CI, just to check the build initiation.
It takes about 5 minutest to run, and shouldn't run if only docs are being changed.

Reply:
sounds good